src/UI/Component/Dropzone/File/Dropzone.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Interface Dropzone
+ *
+ * Describes a dropzone that can be configured to use the darkened background
+ * or not.
```

Comment:
There is may another solution. On the interface a developer can set a background or not. But the color of this background can be configured with less. The method name should not indicate a darkened background, therefore it would better be renamed to:

* `withBackdrop`
* or `withHighligtBackground`
* or `withBackground`

instead of `withDarkenedBackground`.